frontend/src/scenes/surveys/Surveys.tsx

```diff
@@ -37,12 +38,67 @@ import { SurveysDisabledBanner, SurveySettings } from './SurveySettings'
 import { getSurveyStatus, surveysLogic, SurveysTabs } from './surveysLogic'
 
 export const scene: SceneExport = {
-    component: Surveys,
+    component: SurveysWithMaxTool,
     logic: surveysLogic,
     settingSectionId: 'environment-surveys',
 }
 
-export function Surveys(): JSX.Element {
+function SurveysWithMaxTool(): JSX.Element {
+    const {
+        data: { surveys },
+    } = useValues(surveysLogic)
+
+    const { loadSurveys } = useActions(surveysLogic)
+    const { user } = useValues(userLogic)
+
+    return (
+        <MaxTool
+            name="create_survey"
```

Comment:
should we also have a tooltip pointing to our survey templates?

Reply:
sorry, not sure what you mean? like a tooltip, in the Max chat, with our available templates?

Reply:
<img width="342" alt="Screenshot 2025-06-23 at 10 51 57" src="https://github.com/user-attachments/assets/29a1607f-c49e-4a85-af96-3f8917ddd768" />

i meant this but with a message like: check out our templates as well

sometimes they dont need to reinvent the wheel